spec/features/user_deletes_a_spot_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+feature "User deletes a spot" do
+
+  # As a user I want to be able to edit a spot I created
+  # so that the information is up-to-date.
+  # Acceptance criteria:
+  # User can edit the spot they created.
+
+  scenario 'I want to delete a study spot' do
+    skip
```

Comment:
It is so that I can skip this test. We haven't full incorporated the delete function yet.